website/docs/guide/motivation.md

```diff
@@ -0,0 +1,37 @@
+---
+id: motivation
+title: Motivation
+sidebar_label: Motivation
+---
+
+[Solid](https://solidproject.org) is a wonderful project, but the technology it's built on —[RDF](https://en.wikipedia.org/wiki/Resource_Description_Framework)— is unfamiliar to many developers and has a relatively high barrier to entry.
+
+The main goal of lit-pod is to make it as easy as possible for front-end engineers to get started writing web apps that store data on Solid Pods, without hampering the ability to leverage the more advanced capabilities of RDF if so desired.
```

Comment:
```suggestion
The main goal of lit-pod is to make it as easy as possible for engineers to get started writing apps both for the browser and for Node that store data on Solid Pods by providing an abstraction layer on top of both Solid and RDF principles, without hampering the ability to leverage the more advanced capabilities of RDF if so desired.
```
I don't know if this qualifies as documenting for features we don't have, because as long as we don't have a library to log in from Node, the focus is much more on frontend, but technically nothing prevents from using lit-pod in both environments.

Reply:
I'll allow it :P I mean, it already works in Node for public data, and if a Node-compatible fetcher is ever developed, you can use it with lit-pod today, so from our point of view, we definitely support it.